go.mod

```diff
@@ -100,8 +100,8 @@ require (
 	github.com/hashicorp/vault-plugin-auth-jwt v0.13.0
 	github.com/hashicorp/vault-plugin-auth-kerberos v0.6.0
 	github.com/hashicorp/vault-plugin-auth-kubernetes v0.13.0
-	github.com/hashicorp/vault-plugin-auth-oci v0.10.0
-	github.com/hashicorp/vault-plugin-database-couchbase v0.6.0
+	github.com/hashicorp/vault-plugin-auth-oci v0.11.0
+	github.com/hashicorp/vault-plugin-database-couchbase v0.7.0
```

Comment:
Did you mean to include couchbase in this backport? There seems to be a separate backport PR for it already: https://github.com/hashicorp/vault/pull/15619

Reply:
Updated with latest changes in `release/1.11.x` and fixed!